modules-sdks/pax-terminal/instore-solution/NET/CodeExamples/index.md

```diff
@@ -0,0 +1,387 @@
+---
+section: Code Examples
+redirect_from: /modules-sdks/pax-terminal/instore-solution/NET/CodeExamples
+title: C# Code Examples
+description: |
+    Simple code examples without the aim of being state of the art. The code works, so feel free to copy and test.
+menu_order: 1540
+---
+
+## As Client Only
+
+The following is an example of the simplest form of making a program that actually makes a payment. It is a windows form with one button making a payment for 10 SEK and printing the receipt to a TextBox.
```

Comment:
We state 10 SEK as an example but method looks like PAX.PaymentAsync(10); (without currency). Understand that this is not controlled from terminal but maybe we should state "Amount 10 in currency configued for terminal".

Reply:
Well, I guess it says elsewhere that SEK is default. Check out signature of PaymentAsync. Unless the terminal is configured for the supplied currency it won't work.

Reply:
This could of course be changed to always use the currency of the current culture.

Reply:
Note sure what is best, you can decide 👍

Reply:
Is this good to go and can be resolved, or should it be rewritten? :)